src/services/ReportService.ts

```diff
@@ -287,8 +292,21 @@ const submitReport = async (id: number): Promise<Response> => {
 /**
  * upload report photo
  */
-const uploadReportPhoto = async (file: string): Promise<UploadReportPhotoResponse> => {
-  const response: UploadReportPhotoResponse = await httpUploadReportPhoto.post({ entity: { file } });
+const uploadReportPhoto = async (reportId: number, file: File): Promise<UploadReportPhotoResponse> => {
+  const entity = new FormData();
+  entity.append('file', file);
+
+  const headers = {
+    'content-type': 'multipart/form-data',
+  };
+
+  const response: UploadReportPhotoResponse = await httpUploadReportPhoto.post({
+    entity,
+    headers,
+    urlReplacements: {
+      '{reportId}': reportId.toString(),
+    },
+  });
```

Comment:
we could create a PhotoService that does an upload + bulk uploads (maybe it takes in the url as an argument)..  this is a lot of redundant code from withdrawal photo uploads